openfisca_france/parameters/impot_revenu/calcul_impot_revenu/pv/bspce/taux_moins_3_ans.yaml

```
@@ -5,8 +5,9 @@ values:
   1998-01-01:
     value: 0.3
 metadata:
-  short_label: Taux d'imposition pour une activité exercée dans la société depuis moins de 3 ans (case 3SK)
-  1998-01-01:
+  short_label: Taux pour une activité depuis moins de 3 ans
```

```suggestion
  short_label: Taux d'imposition pour une activité exercée depuis moins de 3 ans (case 3SK)
```

J'aurai laissé taux d'imposition, car on n'a plusieurs cas, où ce n'est pas simple de savoir si le taux est un abattement, un taux sur les ressources prises en compte, etc.